include command-line entry points in pyproject.toml (#12)
neutrons/python_project_template: README.md

    8.2 Specify package description at: pyproject.toml#L3

    8.3 Specify package name at: pyproject.toml#L39

    8.4 Specify any terminal entry points (terminal commands) at : pyproject.toml#48. In the example, invoking `packagename-cli` in a terminal is equivalent to running the python script `from packagenamepy.packagename.import main; main()"

>     8.5 We strongly recommended using a single `pyproject.toml` file to manage all the project metadata, including the project name, version, author, license, etc.

Reviewer: ```suggestion
    8.5 Projects will use a  single `pyproject.toml` file to manage all the project metadata, including the project name, version, author, license, etc.
```

>     8.6 Python is moving away from `setup.cfg`/`setup.py`, and we would like to follow the trend for our new projects.

Reviewer: ```suggestion
    8.6 Python has moved away from `setup.cfg`/`setup.py`, and we would like to follow the trend for our new projects.
```
